scripts/zones/Inner_Horutoto_Ruins/mobs/Magicked_Bones.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
+-- Area: Inner Horutoto Ruins
+--  Mob: Magicked Bones
+-----------------------------------
+local ID = zones[xi.zone.INNER_HORUTOTO_RUINS]
+-----------------------------------
+local entity = {}
+
+local function despawn_check(mob)
+    local totd = VanadielTOTD()
+    if totd ~= xi.time.NIGHT and totd ~= xi.time.MIDNIGHT then
+        DespawnMob(mob:getID())
+    end
+end
+
+entity.onMobInitialize = function(mob)
+    if mob:getID() == ID.mob.MAGICKED_BONES_PH_DAGGER + 2 then
+        mob:setModelId(1096) -- Dagger Skeleton Model
```

Comment:
Absolutely not.

Also, I've killed them enough today that I am certain the same ID always gets the same model. just use 2 pool IDs - one for each model.

Reply:
Will look into that then, not as versed in SQL so gotta figure out how to do that properly.